README.md

```diff
@@ -34,14 +34,16 @@ $ zig build
 
     See [Getting older SDKs](#getting-older-sdks) for a guide of how to install additional SDKs for older versions of macOS.
 
-4. Generate deduplicated headers dirs in `<destination>` path
+4. Merge `x86_64` and `aarch64` into destination path `any-macos-any`
```

Comment:
I am thinking, maybe it would be nice to leave docs for the old way with deduping headers in place, perhaps under something like `## Deprecated: abcd` for posterity. What do you think?